src/index.js

```diff
@@ -17,10 +17,23 @@ if (localStorage.getItem('tasks')) {
   /* eslint no-useless-return: "error" */
```

Comment:
- [ OPTIONAL ] I recommend avoiding the use of `eslint disable commands` in your projects, as these errors and warnings are typically helpful for identifying areas where you can improve your code. By resolving these issues, you can write more clean and readable code that adheres to industry standards, which can make it easier to maintain your codebase over time. :100: 

- [ ] In particular, the `eslint no-useless-return` warning indicates that a function contains a return statement that does not actually do anything useful. I suggest resolving this warning by removing any unnecessary or redundant return statements, which can make your code more readable and efficient. :wink: